axisregistry/x_opaque.textproto

```diff
@@ -0,0 +1,13 @@
+# XOPQ based on https://variationsguide.typenetwork.com/#xopq
+tag: "XOPQ"
+display_name: "Y transparent"
+min_value: -1000
+max_value: 2000
+default_value: 88
+precision: 0
+fallback {
```

Comment:
Is fallback required? If so, using the default values seems reasonable. 

Like opsz, parametric axes defaults don't make sense as library-wide/global values, they are all family specific. 

Like Slant, parametric axes have no meaningful global fallbacks.

Reply:
IIRC you have to have at least one fallback so we know what you get on a variable incapable UA

Reply:
Can those be family specific? I also forget...